mindsdb/integrations/handlers/shopify_handler/README.md

```diff
@@ -70,14 +76,16 @@ Watch this video on creating a Shopify access token [here](https://www.youtube.c
 
 ## Example Usage
 
-The first step is to create a database with the new `shopify` engine by passing in the required `shop_url` and `access_token` parameters:
+The first step is to create a database with the new `shopify` engine by passing in the required `shop_url` and `shopify_access_token` parameters. If you are using [Yotpo Product Reviews](https://apps.shopify.com/yotpo-social-reviews) app, you can provide additional keys, `yotpo_app_key` and `yotpo_access_token` (utoken), to access customer reviews:
```

Comment:
Under the **Shopify Handler Initialization** section, please mention these optional parameters.

Reply:
@aditya-azad This does not seem to have been resolved, under the **Shopify Handler Initialization** section, the parameters required to initialize the integration have been given. We should add the new optional parameters here as well.